pages/edge/tutorials/python-flask-server.mdx

```diff
@@ -0,0 +1,235 @@
+import { Steps, Callout, FileTree } from "nextra-theme-docs";
+import { Card, Cards } from "nextra-theme-docs";
+import Install from "@components/install.mdx";
+import Login from "@components/login.mdx";
+import CliVersionCallout from "@components/deploy/CliVersionCallout.mdx";
+
+# Tutorial for deploying a Python Flask Server on Wasmer Edge
+
+In this guide, you'll learn the process of deploying a python flask application on Wasmer Edge.
+We'll be deploying a basic flask server with default index endpoint.
+
+## Deploying a Python Application
+
+<Steps>
+
+<Install />
+
+<CliVersionCallout />
+
+<Login />
+
+### Initialize the Python Application Starter template
+
+```shell copy
+$ wasmer app create
+App type:
+  Static website
+  HTTP server
+  Browser shell
+  JS Worker (experimental)
+> Python Worker
```

Comment:
This should not be worker. Can you open a follow up PR in the wasmer edge cli? We should not say Python worker ever but "Python webserver" instead. Keep this open once that's fixed

Reply:
Yes Can do that. Should we just name it `Python Application` ? Because that'd would serve a more broader prospect ?

Reply:
https://github.com/wasmerio/edge/issues/978

Reply:
Ok, Python Applicaiton is good

Reply:
Looks like this now:
```text
App type:
> Static website
  HTTP server
  Browser shell
  JS Worker (experimental)
  Python Application
```